demisto_sdk/commands/lint/linter.py

```diff
@@ -935,11 +937,8 @@ def _update_support_level(self):
         pack_dir = self._pack_abs_dir.parent if self._pack_abs_dir.parts[-1] == INTEGRATIONS_DIR else \
             self._pack_abs_dir.parent.parent
         pack_metadata_file = pack_dir / PACKS_PACK_META_FILE_NAME
-        with FileLock(f'{pack_metadata_file.as_posix()}.lock'):
-            logger.debug(f'Lock acquired for {pack_metadata_file} to {self._pack_name}')
-            with pack_metadata_file.open() as f:
-                pack_meta_content: Dict = json.load(f)
-        logger.debug(f'Lock released for {pack_metadata_file} of {self._pack_name}')
+        with pack_metadata_file.open() as f:
+            pack_meta_content: Dict = json.load(f)
```

Comment:
Add 2 logger.debug (before and after) messages:
- Before reading content of {f}
- After reading content of {F}